firstvoices/backend/models/page.py

```diff
@@ -40,7 +40,7 @@ class Meta:
     )
     subtitle = models.CharField(blank=True, max_length=constants.DEFAULT_TITLE_LENGTH)
     widgets = models.ForeignKey(
-        SiteWidgetList, on_delete=models.CASCADE, related_name="sitepage_set"
+        SiteWidgetList, on_delete=models.PROTECT, related_name="sitepage_set"
```

Comment:
Should this be SET_NULL instead?

Reply:
The reasoning for having PROTECT here is that in the admin panel if a user tries to delete a SiteWidgetList they will get an error if it is being used by a page. As far as the page APIs go it would work either way and I would be happy to change it.